verify-golint recommends incorrect go get

/kind bug

**What steps did you take and what happened:**
```
$ make                                                                   
go generate ./pkg/... ./cmd/...                        
go fmt ./pkg/... ./cmd/...                                                             
go vet ./pkg/... ./cmd/...                                        
hack/verify-golint.sh                                  
Can not find golint, install with:       
go get -u github.com/golang/lint/golint                
Makefile:52: recipe for target 'lint' failed
make: *** [lint] Error 1
```

**What did you expect to happen:**
go get golang.org/x/lint/golint
per https://github.com/golang/lint/issues/415

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/hack/verify-golint.sh
+++ b/hack/verify-golint.sh
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
-  echo 'go get -u github.com/golang/lint/golint'
+  echo 'go get -u go get golang.org/x/lint/golint'
   exit 1
 fi
 

--- a/hack/verify-golint.sh
+++ b/hack/verify-golint.sh
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
-  echo 'go get -u go get golang.org/x/lint/golint'
+  echo 'go get -u golang.org/x/lint/golint'
   exit 1
 fi
 

